pkg/osvscanner/osvscanner.go

```diff
@@ -815,7 +849,7 @@ func DoScan(actions ScannerActions, r reporter.Reporter) (models.VulnerabilityRe
 
 	for _, dir := range actions.DirectoryPaths {
 		r.Infof("Scanning dir %s\n", dir)
-		pkgs, err := scanDir(r, dir, actions.SkipGit, actions.Recursive, !actions.NoIgnore, actions.CompareOffline)
+		pkgs, err := scanDir(r, dir, actions.SkipGit, actions.Recursive, !actions.NoIgnore, actions.CompareLocally)
```

Comment:
This should not be changed, as comparing locally is different from comparing offline - only the latter requires the former

Reply:
replied in another comment.